plugins/ckan_operators/resource_operator.py

```diff
@@ -96,11 +229,9 @@ def execute(self, context):
             )
 
         try:
-            file_last_modified = download_file_info["last_modified"])
+            file_last_modified = parser.parse(download_file_info["last_modified"])
```

Comment:
All of these changes below relating to the ResourceAndFileOperator can be ignored, like the ones above relating to the timestamp